docs/Parallel-Learning-Guide.rst

```diff
@@ -1,24 +1,19 @@
-Parallel Learning Guide
-=======================
+Distributed Learning Guide
+==========================
+
+.. _Parallel Learning Guide:
 
 This guide describes distributed learning in LightGBM. Distributed learning allows the use of multiple machines to produce a single model.
 
 Follow the `Quick Start <./Quick-Start.rst>`__ to know how to use LightGBM first.
 
-**List of external libraries in which LightGBM can be used in a distributed fashion**
-
--  `Dask API of LightGBM <./Python-API.rst#dask-api>`__ (formerly it was a separate package) allows to create ML workflow on Dask distributed data structures.
+How Distributed LightGBM Works
+------------------------------
 
--  `MMLSpark`_ integrates LightGBM into Apache Spark ecosystem.
-   `The following example`_ demonstrates how easy it's possible to utilize the great power of Spark.
-
--  `Kubeflow Fairing`_ suggests using LightGBM in a Kubernetes cluster.
-   `These examples`_ help to get started with LightGBM in a hybrid cloud environment.
-   Also you can use `Kubeflow XGBoost Operator`_ to train LightGBM model.
-   Please check `this example`_ for how to do this.
+This section describes how distributed learning in LightGBM works. To learn how to do this in various programming languages and frameworks, please see `Integrations <#integrations>`_.
```

Comment:
This link doesn't work.

Reply:
For some reason it is `#id1`!..

Reply:
oh weird, didn't notice that! I'll have to experiment and see if I can fix it

Reply:
OK, I see now! Should work now.
```suggestion
This section describes how distributed learning in LightGBM works. To learn how to do this in various programming languages and frameworks, please see `Integrations <#integrations>`__.
```